### ISSUE TITLE
Failed to load the session on Linux

### ISSUE THREAD

**Issue by talentlessguy** [radicle-dev/radicle-upstream#2181]
Error message:

```json
{
    "stack": "Error: Failed to load the session\n    at cn (file:///tmp/.mount_radiclr4EFsN/resources/app.asar/public/bundle.js:1:1915695)\n    at async file:///tmp/.mount_radiclr4EFsN/resources/app.asar/public/bundle.js:35:120417",
    "message": "Failed to load the session",
    "code": "SessionFetchFailure",
    "source": {
      "stack": "TypeError: Failed to fetch",
      "message": "Failed to fetch",
      "code": "UnknownException",
      "details": {
        "name": "TypeError"
      }
    }
  }
```

![image](https://user-images.githubusercontent.com/35937217/126911387-6d21a758-c182-4196-b275-f98ea6f8ff7e.png)

AppImage logs (IPs are hidden):

```
Jul 25 22:44:45.431  INFO api::process: starting API
Jul 25 22:44:45.433  INFO api::process: starting peer
Jul 25 22:44:45.443  INFO ifwatch{bound_addr=0.0.0.0:48502}: librad::net::quic::endpoint: adding listen addr 127.0.0.1:48502
Jul 25 22:44:45.443  INFO ifwatch{bound_addr=0.0.0.0:48502}: librad::net::quic::endpoint: adding listen addr 192.168.0.106:48502
Jul 25 22:44:45.444  INFO accept{peer_id=hydntk87mtiqrw11qdxc6f7wx4c4txpdwec19ici8zptjqy9s9wzfy}:disco:discovered{remote_id=hynkyndc6w3p8urucakobzna7sxwgcqny7xxtw88dtx3pkf7m3nrzc}:connect{remote_id=hynkyndc6w3p8urucakobzna7sxwgcqny7xxtw88dtx3pkf7m3nrzc}: librad::net::protocol::io::connections: establishing connection remote_addr=HIDDEN_IP
Jul 25 22:44:45.545  INFO accept{peer_id=hydntk87mtiqrw11qdxc6f7wx4c4txpdwec19ici8zptjqy9s9wzfy}:disco:discovered{remote_id=hynewpywqj6x4mxgj7sojhue3erucyexiyhobxx4du9w66hxhbfqbw}:connect{remote_id=hynewpywqj6x4mxgj7sojhue3erucyexiyhobxx4du9w66hxhbfqbw}: librad::net::protocol::io::connections: establishing connection remote_addr=HIDDEN_IP
Jul 25 22:44:45.597  WARN accept{peer_id=hydntk87mtiqrw11qdxc6f7wx4c4txpdwec19ici8zptjqy9s9wzfy}:disco:discovered{remote_id=hynewpywqj6x4mxgj7sojhue3erucyexiyhobxx4du9w66hxhbfqbw}:connect{remote_id=hynewpywqj6x4mxgj7sojhue3erucyexiyhobxx4du9w66hxhbfqbw}: librad::net::protocol::io::connections: could not connect err=Connection(ConnectionClosed(ConnectionClose { error_code: Code::crypto(78), frame_type: None, reason: b"peer doesn't support any known protocol" })) remote_addr=HIDDEN_IP
Jul 25 22:44:45.597  INFO accept{peer_id=hydntk87mtiqrw11qdxc6f7wx4c4txpdwec19ici8zptjqy9s9wzfy}:disco:discovered{remote_id=hyd1i3rrnm685uj9ji9ihmmsaiy8nedo77o11c7pqxdx96z8o47f4q}:connect{remote_id=hyd1i3rrnm685uj9ji9ihmmsaiy8nedo77o11c7pqxdx96z8o47f4q}: librad::net::protocol::io::connections: establishing connection remote_addr=HIDDEN_IP
```

temporary fix: remove all radicle folders

**Comment by geigerzaehler** [comment]
This is odd :thinking:. Although the API is starting it looks like the app can’t connect to it. Could you open the dev tools in the app (Ctrl+i or Cmd+i) and check the “Network” tab? I’m interested in the error that happens when connecting to `http://localhost:17246`—they should tell us more.

**Comment by talentlessguy** [comment]
@geigerzaehler this is not reproducible anymore :((( after deleting the data folders it worked fine again

**Comment by talentlessguy** [comment]
nevermind it happened again lol

![image](https://user-images.githubusercontent.com/35937217/126965770-6ce71bbe-69e3-4e13-bc40-27a28a974599.png)

gonna send devtools result

**Comment by talentlessguy** [comment]
devtools:

![image](https://user-images.githubusercontent.com/35937217/126965852-dafad78c-eeda-4915-95d7-984f034a0b89.png)

**Comment by geigerzaehler** [comment]
After you start the app, can you run the following from your terminal and post the output?
```bash
curl -v http://localhost:17246
curl -v http://127.0.0.1:17246
```

**Comment by talentlessguy** [comment]
@geigerzaehler this is what i have:

```
➜ curl -v http://localhost:17246
  curl -v http://127.0.0.1:17246
*   Trying ::1:17246...
* connect to ::1 port 17246 failed: Connection refused
*   Trying 127.0.0.1:17246...
* Connected to localhost (127.0.0.1) port 17246 (#0)
> GET / HTTP/1.1
> Host: localhost:17246
> User-Agent: curl/7.77.0
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< content-type: application/json
< content-length: 54
< date: Wed, 28 Jul 2021 10:00:49 GMT
< 
* Connection #0 to host localhost left intact
{"message":"Resource not found","variant":"NOT_FOUND"}*   Trying 127.0.0.1:17246...
* Connected to 127.0.0.1 (127.0.0.1) port 17246 (#0)
> GET / HTTP/1.1
> Host: 127.0.0.1:17246
> User-Agent: curl/7.77.0
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< content-type: application/json
< content-length: 54
< date: Wed, 28 Jul 2021 10:00:49 GMT
< 
* Connection #0 to host 127.0.0.1 left intact
{"message":"Resource not found","variant":"NOT_FOUND"}⏎     
```

**Comment by geigerzaehler** [comment]
I suspect that the app tries to connect to the proxy via IPv6 while the proxy is only available on the IPv4 interface. We’ll fix this in #2195.

**Comment by talentlessguy** [comment]
@geigerzaehler nice to hear that! thanks for investigating the issue

looking forward to next release
